Dockerfile-k8s

```diff
@@ -0,0 +1,15 @@
+FROM clojure:lein-2.8.1-alpine
+
+RUN adduser -S -D -h /nonexistent -H service_user
+
+RUN apk --no-cache add libwebp libwebp-tools imagemagick bash
```

Comment:
imagemagick with explicit version should allow easy rebuild image in future, there're many cases when we want to upgrade just imagemagick.